File: Python/Logic - 1/alarm_clock.py
def alarm_clock(day, vacation):
    
    weekdays = [1,2,3,4,5]
    
    weekends = [0,6]
    
    if vacation:
        
        if day in weekdays:
            
            return '10:00'
        
        else:
            
            return 'off'
        
    else:
        
        if day in weekdays:
            
            return '7:00'
        
        else:
            
            return '10:00'

File: Java/Logic - 1/alarmClock.java
public String alarmClock(int day, boolean vacation) {

    if(day>0&&day<6) {

        if(vacation==true) {

            return "10:00";

        }

        else {

            return "7:00";

        }

    }

    else {

        if(vacation==true){

            return "off";

        }

        else {

            return "10:00";

        }

    }

}